crates/holochain/src/conductor/cell.rs

```diff
@@ -440,6 +440,7 @@ impl Cell {
     }
 
     /// a remote node is attempting to retrieve a validation package
+    #[tracing::instrument(skip(self))]
```

Comment:
```suggestion
    #[tracing::instrument(skip(self))]
    #[tracing::instrument(skip(self), level = "trace")]
```
This is also a hot path see the comment below